include/global.h

```diff
@@ -115,6 +114,7 @@
 
 #define ROUND_BITS_TO_BYTES(numBits)(((numBits) / 8) + (((numBits) % 8) ? 1 : 0))
 
+#define POKEMON_SLOTS_NUMBER 810
```

Comment:
I think this should be defined in constants/species.h and/or changed to reflect the NUM_SPECIES change from pret

Reply:
It was originally defined right below the floating point stuff, and placing it here avoids a merge conflict in BE-PE merges.

Reply:
Also in PE, `NUM_SPECIES != POKEMON_SLOTS_NUMBER` because it is used differently